src/node_util.cc

```diff
@@ -186,7 +188,63 @@ void Initialize(Local<Object> target,
   env->SetMethod(target, "safeGetenv", SafeGetenv);
 }
 
+void InitializeUnsafe(Local<Object> target,
+                      Local<Value> unused,
+                      Local<Context> context) {
+  InitializeBase(target, unused, context);
+}
+
+void PromiseHook(PromiseHookType type,
+                 Local<Promise> promise,
+                 Local<Value> value,
+                 void* arg) {
+  switch (type) {
```

Comment:
Here this uses a switch, elsewhere if-elseif statements are used. Can it be consistent?

Reply:
the switch here is used to waterfall behaviour through multiple constants whereas above the if statements are for mapping constants.